.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         - 
           qt_version: "5.15.2"
           modules: ""
-          os: ubuntu-18.04
+          os: ubuntu-latest
         - 
           qt_version: "6.2.4"
```

Comment:
`qt_version: "6.3.1"` - again, this seems to work for me.

Reply:
I plan on adding 6.3.1 as well soon. My plan is to at least build on one LTS version (6.2.4 in this case) along with the latest available Qt version. I'll keep 5.15 for now due to its extended LTS, but if it ever starts causing pains to develop and support it (work arounds or lots of `#ifdefs` then it will be dropped).

Reply:
Are you sure binaries made on Ubuntu 22.04 will run on 20.04? Are you requiring a newer ubuntu?

Reply:
> Are you sure binaries made on Ubuntu 22.04 will run on 20.04?

The short answer is: no.

That's the main reason for this draft PR so that if there are cases like this that I'm unaware of then we can make sure that this is accounted for. If there is a known justification to not use the latest ubuntu version then I'm perfectly fine with that.